controllers/keywordController.js

```diff
@@ -21,25 +21,32 @@ const create = async (req, res) => {
     }
   }
 
-  const isDuplicatedKeyword =
-    (await keywordModel.findOne({ groupId: req.body.groupId, keywordList: req.body.keyword })) ===
-    null;
+  const isDuplicatedKeyword = await groupModel
+    .findOne({ _id: req.body.groupId })
+    .populate("keywordIdList", "keyword")
+    .exec()
+    .then((query) => {
+      return query.keywordIdList.some((doc) => doc.keyword === req.body.keyword);
+    })
+    .catch((error) => {
+      return res.status(400).send({ message: "[InvalidGroupId] Error occured" });
+    });
```

Comment:
```suggestion
    .catch(() => {
      return res.status(400).send({ message: "[InvalidGroupId] Error occured" });
    });
```
catch 뒤 불필요한 `error`로 보여 지우도록 하겠습니다!